src/api/db/student_course_selection.py

```diff
@@ -1,45 +1,49 @@
-class student_course_selection:
-	def __init__(self, db_conn):
-		self.db_conn = db_conn
+from db.model import *
+from models.student_course_selection import StudentCourseSelection
 
-	def add_selection(self, name, sem, uid, cid):
+
+class student_course_selection(Model):
+	def __init__(self):
+		super().__init__()
+
+	async def add_selection(self, name, sem, uid, cid):
 		sql = 	"""
 				INSERT INTO
 					student_course_selection (user_id, semester, course_name, crn)
 				VALUES
-					(%s, %s, %s, %s)
+					('%s', '%s', '%s', '%s')
 				ON CONFLICT DO NOTHING;
 				"""
-		resp, error = self.db_conn.execute(sql, [uid, sem, name, cid], False)
+		resp, error = await self.db_conn.execute(sql, [uid, sem, name, cid], False)
 		return (True, None) if not error else (False, error)
 
-	def remove_selection(self,name,sem,uid,cid):
+	async def remove_selection(self, name, sem, uid, cid):
 		if cid is None:
 			sql = 	"""
 					DELETE FROM
 						student_course_selection
 					WHERE
-						user_id = %s AND
-						semester = %s AND
-						course_name = %s
+						user_id = '%s' AND
+						semester = '%s' AND
+						course_name = '%s'
 					"""
 			resp, error = self.db_conn.execute(sql, [uid, sem, name], False)
 		else:
 			sql = 	"""
 					DELETE FROM
 						student_course_selection
 					WHERE
-						user_id = %s AND
-						semester = %s AND
-						course_name = %s AND
-						crn = %s
+						user_id = '%s' AND
+						semester = '%s' AND
+						course_name = '%s' AND
```

Comment:
I could be wrong but I believe its possible for course names to have single quotes within them. We should probably escape this with the $<tag>$...$<tag>$ syntax as a precaution. You can see examples of this in the new db/courses.py file.